### ISSUE TITLE
In-between words space detection/signaling if spaces arew wider than 1 character

### ISSUE THREAD

**Issue by bhlevca** [ONLYOFFICE/DesktopEditors#943]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Is there any setting in DesktopEditors that would find/underline/highlight in-between words spaces that are more than 1 character wide? 

MS Office has it and it is a very useful feature when editing formal documents like publishing articles.
It should not be too difficult to implement while using the spelling module.

**Comment by Gingetsu** [comment]
Hello, @bhlevca.
Try entering two special characters "^w^w" into the search tool.
Thank you for report.

**Comment by bhlevca** [comment]
if ^W means CTRL+W then it does not work on Linux+KDE. CTRL+W is mapped to a window function.

**Comment by ShockwaveNN** [comment]
^w literally means to input those symbols, without any key combinations

![image](https://user-images.githubusercontent.com/668524/178683205-d4971944-a0c7-4157-9caa-7f565993cbdf.png)

**Comment by bhlevca** [comment]
YES, this way it works.  It would be good if these features can be documented in a manual or help files

**Comment by ShockwaveNN** [comment]
It is [documented](https://helpcenter.onlyoffice.com/onlyoffice-editors/onlyoffice-document-editor/helpfulhints/search.aspx)

**Comment by bhlevca** [comment]
Thanks for the link. Is indeed there, but some how hidden and if you don't expand the link you can miss it. Thank again. An interactive search/help withing the application that would lead to the page would be ideal.

**Comment by Gingetsu** [comment]
@bhlevca, we know about this behaviour. We discuss this issue(list-a hint in the search) with our development team (issue 56516 in our internal issue tracker).

**Comment by ShockwaveNN** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too
